cluster/gce/gci/configure-helper.sh

```diff
@@ -597,6 +597,9 @@ function create-master-auth {
   if [[ -n "${GCE_GLBC_TOKEN:-}" ]]; then
     append_or_replace_prefixed_line "${known_tokens_csv}" "${GCE_GLBC_TOKEN},"                "system:controller:glbc,uid:system:controller:glbc"
   fi
+  if [[ -n "${ADDON_MANAGER_TOKEN:-}" ]]; then
+    append_or_replace_prefixed_line "${known_tokens_csv}" "${ADDON_MANAGER_TOKEN},"   "system:addon-manager,admin,system:masters"
```

Comment:
The entry after the "Name" is the "UID." It's mostly not used, but setting it to "admin" is confusing. Make it `uid:system:addon-manager`.